packages/app/lib/carousel/carousel.tsx

```diff
@@ -10,23 +10,24 @@ import RNRCarousel from "react-native-reanimated-carousel";
 import type { ICarouselInstance } from "react-native-reanimated-carousel";
 
 import { useIsDarkMode } from "@showtime-xyz/universal.hooks";
-import { colors } from "@showtime-xyz/universal.tailwind";
+import { colors, styled } from "@showtime-xyz/universal.tailwind";
 import { View } from "@showtime-xyz/universal.view";
 
-import { Controller } from "./controller";
 import { CarouselProps } from "./types";
 
+const StyledRNRCarousel = styled(RNRCarousel);
 export function Carousel({
```

Comment:
I think making this `memo` might help with perf, as long as all passed props are stable as well